src/jdbc/cubrid/jdbc/net/BrokerHandler.java

```diff
@@ -227,6 +228,20 @@ public static int statusBroker(String ip, int port, int process, byte[] session,
         return status;
     }
 
+    public static boolean isValid(String ip, int port, int process, byte[] session, int timeout)
```

Comment:
This function just wraps BrokerHandler.statusBroker(). I think we don't need to declare isValid () in the BrokerHandler class. It looks not related to other functions in the same class.